### DISCUSSION TITLE
DAAN训练相关问题

### DISCUSSION THREAD

**Discussion by yuCodeNature** [jindongwang/transferlearning#422] [Q&A]
王老师您好，我这边使用了您的DAAN的代码，数据集换成了轴承振动信号小波变换后的图集，训练时发现效果不是很好，其中global_Loss和local_Loss这两个参数训练过程中几乎没有变化（global_Loss:0.0773, local_Loss:0.0968上下浮动），请问您有什么头绪吗，谢谢您

**Comment by PaulinaLeszczelowska** [comment]
did you manage to fix this?
